docker/test-images/zipkin-opensearch2/start-opensearch

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+#
+# Copyright The OpenZipkin Authors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# ENTRYPOINT script that starts OpenSearch
+#
+# This intentionally locates config using the current working directory, in order to consolidate
+# Dockerfile instructions to WORKDIR
+set -eu
+
+# Configure the Docker HEALTHCHECK
+export HEALTHCHECK_PORT=9200
+export HEALTHCHECK_PATH=/_cluster/health
+
+exec ./bin/opensearch
```

Comment:
in all our other java images, including elasticsearch, we use the java commands directly. This prevents us to have to install things like bash, or accidentally have zombie or otherwise issues when the entrypoint has several processes below it. Can you try to do the same here?

Reply:
most notably this gives us JRE flexibility and transparency. As this is a test image, we want to be able to update the JRE even past what's formally specified sometimes. When the java args are here we get an advantage to modify to apply things like a missing open or whatever. One way to figure out what's going on is to run the normal docker image then `docker exec` into it and use PS or whatever if pid 1 isn't java. Hope this helps!

Reply:
Sure, I will take some time though (I've tried that inititially but got dragged into tons of startup issues)

Reply:
ack, now you know the labor I've done for many years ;)

Reply:
good news is typically the hardest is the first one. For example, ES was mechanical until 8.x when a lot of things changed. anyway thanks for having a look, as it saves me from doing the same.

Reply:
should be all set here, @codefromthecrypt !